### ISSUE TITLE
p. 67とp. 68の図で「int_dotimes」と「"times"」というように表記が違う

### ISSUE THREAD

**Issue by kou** [ohmsha/ruby-under-a-microscope-ja#8]
p. 67の図の「int_dotimes」が正そう。

**Comment by snoozer05** [comment]
原文からこのようになっていて、おそらく読者にわかりやすいようにこうしているのではと思います。どうするか考えますね。

**Comment by patshaughnessy** [comment]
Yes, I would change Figure 3-12 to use "int_dotimes" to make it consistent with Figure 3-11. Figure 3-11 is correct.

**Comment by snoozer05** [comment]
I would change Figure 3-12 of Japanese Edition to use "int_dotimes" too :)

**Comment by hisashim** [comment]
@patshaughnessy Thanks!
@snoozer05 I'll edit the image for Figure 3-12.
